File: lua/raindrops/raindrops.lua
return function(n)
  local res = ""

  if n % 3 == 0 then
    res = res .. "Pling"
  end
  if n % 5 == 0 then
    res = res .. "Plang"
  end

  if n % 7 == 0 then
    res = res .. "Plong"
  end

  if res == "" then
    res = tostring(n)
  end

  return res
end

File: go/raindrops/raindrops.go
// Package raindrops
package raindrops

import (
	"strconv"
)

func Convert(number int) string {
	res := ""
	if number%3 == 0 {
		res = res + "Pling"
	}
	if number%5 == 0 {
		res = res + "Plang"
	}
	if number%7 == 0 {
		res = res + "Plong"
	}

	if res != "" {
		return res
	} else {
		res = strconv.Itoa(number)
		return res
	}
}
